packages/mdc-textfield/foundation.js

```diff
@@ -142,6 +150,24 @@ class MDCTextFieldFoundation extends MDCFoundation {
     this.receivedUserInput_ = true;
   }
 
+  /**
+   * Updates the focus outline for outlined text fields.
+   */
+  updateOutline() {
+    if (!this.outline_) {
+      return;
+    }
+    const width = this.adapter_.getWidth();
+    const height = this.adapter_.getHeight();
+    // The label is scaled 75% when it floats above the text field.
+    const labelWidth = this.adapter_.getLabelWidth() * 0.75;
+    const radiusStyleValue = this.adapter_.getIdleOutlineStyleValue('border-radius') ||
+      /* Firefox requires specifying the corner. */ this.adapter_.getIdleOutlineStyleValue('border-top-left-radius');
```

Comment:
Maybe move this to a line comment above this statement to make the code flow better? e.g. // Fall back to reading a specific corner's style because Firefox doesn't report the style on border-radius"